base/linalg/blas.jl

```diff
@@ -472,22 +524,24 @@ end
 """
     gemv!(tA, alpha, A, x, beta, y)
 
-Update the vector `y` as `alpha*A*x + beta*y` or `alpha*A'x + beta*y` according to `tA`
-(transpose `A`). Returns the updated `y`.
+Update the vector `y` as `alpha*A*x + beta*y` or `alpha*A'x + beta*y`
+according to [`tA`](@ref stdlib-blas-trans).
+`alpha` and `beta` are scalars. Returns the updated `y`.
 """
 gemv!
 
 """
     gemv(tA, alpha, A, x)
 
-Returns `alpha*A*x` or `alpha*A'x` according to `tA` (transpose `A`).
+Returns `alpha*A*x` or `alpha*A'x` according to [`tA`](@ref stdlib-blas-trans).
```

Comment:
`` [`tA`](@ref stdlib-blas-trans) `` is going to cross reference the `-` docstring rather than `@id stdlib-blas-trans` due to the way Documenter currently resolves `@ref`. A `@ref`d code literal will search for a docstring, rather than a header, to link to and the expression `stdlib-blas-trans` looks like a `-` method call to it.

I see whether it can be changed to act a bit more intelligently here.